File: emulinker/src/main/java/org/emulinker/util/EmuLinkerPropertiesConfig.java
package org.emulinker.util;

import org.apache.commons.configuration.ConfigurationException;
import org.apache.commons.configuration.PropertiesConfiguration;

public class EmuLinkerPropertiesConfig extends PropertiesConfiguration {

  public EmuLinkerPropertiesConfig() throws ConfigurationException {
    super(EmuLinkerPropertiesConfig.class.getResource("/emulinker.cfg"));
    setThrowExceptionOnMissing(true);
  }
}

File: emulinker/src/main/java/org/emulinker/util/EmuLinkerPropertiesConfig.kt
package org.emulinker.util

import org.apache.commons.configuration.PropertiesConfiguration

class EmuLinkerPropertiesConfig :
    PropertiesConfiguration(EmuLinkerPropertiesConfig::class.java.getResource("/emulinker.cfg")) {
  init {
    isThrowExceptionOnMissing = true
  }
}
